docs/ssss.md

```diff
@@ -1,3 +1,94 @@
 # Simperby Super Simple Summary
 
-TODO
+## DAO
+
+A DAO (Decentralized Autonomous Organization) is an organization that runs on a
+blockchain, governed by rules set in a contract, and operates autonomously
```

Comment:
What do you mean by "a contract"? It sounds like that a DAO must implemented as a smart contract, while Simperby isn't.

Reply:
Consider rewriting to

> A DAO (Decentralized Autonomous Organization) is an organization that operates autonomously on a blockchain without government backing, governed by ...

Reply:
Fixed.